ThingIFSDK/ThingIFSDK/Trait/TraitCommandForm.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TraitCommandForm.swift
+//  ThingIFSDK
+//
+//  Created on 2016/12/08.
+//  Copyright (c) 2016 Kii. All rights reserved.
+//
+
+import Foundation
+
+/**
+Form of a command for trait version thing.
+
+This class contains data in order to create `Command` with
```

Comment:
https://github.com/KiiPlatform/thing-if-AndroidSDK/pull/152#issuecomment-265926717

I misunderstand a specification. I will fix this.